src/rust/inetstack/protocols/tcp/active_open.rs

```diff
@@ -271,32 +256,6 @@ impl SharedActiveOpenSocket {
         // Start connection handshake.
         let handshake_retries: usize = self.tcp_config.get_handshake_retries();
         let handshake_timeout = self.tcp_config.get_handshake_timeout();
-        // Look up remote MAC address.
```

Comment:
This code has been moved to a centralized place. I believe it was originally suppose to replicate the retry mechanism within the ARP cache but is actually off by one retry compared to that code.